etc/profile-m-z/tidal-hifi.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for tidal-hifi
+# Description: The web version of Tidal running in electron with hifi support thanks to widevine.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include tidal-hifi.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}.cache/AMD/VkCache
+noblacklist ${HOME}.cache/fontconfig
+noblacklist ${HOME}.cache/mesa_shader_cache
+noblacklist ${HOME}.config/Electron
+noblacklist ${HOME}.config/pulse
+noblacklist ${HOME}.config/tidal-hifi
+
+include disable-common.inc
+include disable-devel.inc
+#include disable-exec.inc # breaks widevine
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+mkdir ${HOME}.cache/AMD/VkCache
+mkdir ${HOME}.cache/fontconfig
+mkdir ${HOME}.cache/mesa_shader_cache
+mkdir ${HOME}.config/Electron
+mkdir ${HOME}.config/pulse
+mkdir ${HOME}.config/tidal-hifi
+whitelist ${HOME}.cache/AMD/VkCache
+whitelist ${HOME}.cache/fontconfig
+whitelist ${HOME}.cache/mesa_shader_cache
+whitelist ${HOME}.config/Electron
+whitelist ${HOME}.config/pulse
+whitelist ${HOME}.config/tidal-hifi
+
+apparmor
```

Comment:
Breaks widevine

Reply:
I tested this and it works for me, not sure if this might be different for others.

Reply:
Do you have AA enabled? In 1. firejail 2. your kernel 3. loaded the firejail AA profile?

Reply:
Yes, checked everywhere and apparmor is enabled, and I see `AppArmor enabled` in the debug log. I am seeing:
```
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
```
however I'm still able to stream at the highest bitrate and access all music, videos, etc.

Reply:
Because it should `noexec ${HOME}` as well. Nevermind, if it works.